examples/contactnets_simple.py

```diff
@@ -42,8 +42,11 @@
 ELBOW_URDFS = {MESH_TYPE: ELBOW_MESH_URDF_ASSET, BOX_TYPE: ELBOW_BOX_URDF_ASSET}
 URDFS = {CUBE_SYSTEM: CUBE_URDFS, ELBOW_SYSTEM: ELBOW_URDFS}
 
-STORAGE_NAME = os.path.join(os.path.dirname(__file__), 'storage',
-                            CUBE_DATA_ASSET)
+DATA_STORAGE_NAME = os.path.join(os.path.dirname(__file__), 'storage',
```

Comment:
`DATA_STORAGE_NAME` and `RESULTS_STORAGE_NAME` aren't being used in this test script.

Reply:
Removed.